chutney/server-core/src/main/java/com/chutneytesting/server/core/domain/execution/ExecutionRequest.java

```diff
@@ -36,27 +37,28 @@ public class ExecutionRequest {
     public final CampaignExecution campaignExecution;
     public final Set<String> tags;
 
-    public ExecutionRequest(TestCase testCase, String environment, String userId, DataSet dataset, CampaignExecution campaignExecution) {
+    public ExecutionRequest(TestCase testCase, String environment, String userId, DataSet dataset, CampaignExecution campaignExecution, Set<String> tags) {
```

Comment:
I would have kept the constructor without the tags parameter and rename the tags parameter to extraTags to emphasize the fact that tags come from elsewhere too.